sc-memory/sc-memory/sc_memory.cpp

```diff
@@ -165,6 +170,12 @@ void ScMemoryContext::Destroy()
   }
 }
 
+ScAddr ScMemoryContext::GetUserAddr()
+{
+  CHECK_CONTEXT;
+  return sc_memory_context_get_user_addr(m_context);
+}
+
 void ScMemoryContext::BeingEventsPending()
```

Comment:
BeginEventsPending*

Reply:
It had better to fix it in https://github.com/ostis-ai/sc-machine/pull/371